### PR TITLE
Add an accept_licenses option to pkg5 module

### PR DESCRIPTION
**Pull request by mavit**
This accepts any software licences required by the package.

### CODE REVIEW

File: packaging/os/pkg5.py [hunk 1]
```diff
@@ -39,6 +39,13 @@
     required: false
     default: present
     choices: [ present, latest, absent ]
+  accept:
+    description:
+      - Accept any licences.
+    default: false
+    required: false
+    default: false
```
Comment (bcoca) [inline packaging/os/pkg5.py:46]: extra default

File: packaging/os/pkg5.py [hunk 2]
```diff
@@ -39,6 +39,13 @@
     required: false
     default: present
     choices: [ present, latest, absent ]
+  accept:
```
Comment (bcoca) [inline packaging/os/pkg5.py]: can we rename to accept_licenses? you can keep accept as an alias

**Comment by resmo** [comment]
and there is a typo in the commit msg

**Comment by mavit** [comment]
- Option renamed to `accept_licenses`.
- Duplicate `default` removed from docs.
- Typo corrected in commit message.
- Replaced changes missing due to a bad merge.

**Pull request merged**